news/2024/2024-12-05-osu-world-cup-2024-finals-recap.md

```diff
@@ -0,0 +1,117 @@
+
+## Finals summary
+
+| Team A |  |  | Team B |
+| --: | :-: | :-: | :-- |
+| **Australia** ::{ flag=AU }:: | **7** | 2 | ::{ flag=RU }:: Russian Federation |
+| **Germany** ::{ flag=DE }:: | **7** | 3 | ::{ flag=GB }:: United Kingdom |
+| **South Korea** ::{ flag=KR }:: | **7** | 5 | ::{ flag=US }:: United States |
+| **Australia** ::{ flag=AU }:: | **7** | 3 | ::{ flag=DE }:: Germany |
+
+In the lower bracket, four teams, each with their own story to tell, sought the rarefied air of a podium spot. Australia, looking to retain their place among the top 3 — Germany, seeking a return to the glory days of 2021 and 2022 — Russia, finally having broken their so-called "semifinals curse" and making it into the top 6 for the first time since 2019 — and the United Kingdom, vying to re-establish themselves among the upper echelon of the osu!standard scene.
+
+The week opened with Australia facing off against Russia. Both teams came out firing with their first picks, but it would be Australia to muster up the first break point of the match on Russia's second pick, [Blue Night](https://osu.ppy.sh/beatmapsets/31333#osu/116006), and afterwards they never looked back. Australia dropped only one more point the rest of the way, a narrow Russian victory on [Dear The Night I Loved](https://osu.ppy.sh/beatmapsets/2284389#osu/4872029), en route to a 7–2 victory. Australia moved on to the 4th place match, while Russia was sent home.
+
+Match number two of the week pitted old foes Germany and the United Kingdom against each other once more. With ::{ flag=DE }:: [criller](https://osu.ppy.sh/users/8116659), ::{ flag=DE }:: [WhiteCat](https://osu.ppy.sh/users/4504101), and the rest of the team performing like a well-oiled machine, Germany raced to a 6–0 lead in this match. The United Kingdom, despite riding momentum from their incredible tiebreaker victory over Canada in the previous round, appeared to have met their match, coming up short on map after map. However, a 3-way S rank on [Dear The Night I Loved](https://osu.ppy.sh/beatmapsets/2284389#osu/4872029) won them that map by a mere 54,882 points, and they followed it up with another miniscule victory by 88,393 on [Horobi o Inorishi Mono](https://osu.ppy.sh/beatmapsets/2284565#osu/4872576), then took [SILVERY](https://osu.ppy.sh/beatmapsets/1581365#osu/3229080), the anti-mod pick, by over 200k.
+
+Some thought a historic reverse sweep was in the making, but Germany shattered those dreams, opting into [Eiyuu no Tsurugi](https://osu.ppy.sh/beatmapsets/1864097#osu/3833517) and ending the match with a tight margin of 88,814 points, despite ::{ flag=GB }:: [Bubbleman](https://osu.ppy.sh/users/5182050)'s best efforts with a lobby-best 716k score. It was on to Australia and a shot at the podium for Germany, while the United Kingdom were eliminated with a top 6 placement for the first time since 2021.
```

Comment:
```suggestion
Some thought a historic reverse sweep was in the making, but Germany shattered those dreams, opting into [Eiyuu no Tsurugi](https://osu.ppy.sh/beatmapsets/1864097#osu/3833517) and ending the match with a tight margin of 88,814 points, despite ::{ flag=GB }:: [Bubbleman](https://osu.ppy.sh/users/5182050)'s best efforts with a lobby-best 716k score. It was on to Australia and a shot at the podium for Germany, while the United Kingdom was eliminated with a top 6 placement for the first time since 2021.
```